src/__tests__/autocapture.js

```diff
@@ -727,14 +735,57 @@ describe('Autocapture system', () => {
             autocapture._captureEvent({ target: span, type: 'click' }, lib)
             expect(lib.capture.callCount).toBe(0)
         })
+
+        it('does not capture any input values if mask_all_inputs is set', () => {
+            const dom = `
+      <button id='button1'>
+        Not sensitive
+      </button>
+      `
+            const sandbox = sinon.createSandbox()
+
+            const lib = {
+                _ceElementTextProperties: [],
+                get_distinct_id() {
+                    return 'distinctid'
+                },
+                capture: sandbox.spy(),
+                get_config: sandbox.spy(function (key) {
+                    switch (key) {
+                        case 'mask_all_inputs':
+                            return false
+                    }
+                }),
+            }
+
+            document.body.innerHTML = dom
+            const button1 = document.getElementById('button1')
+
+            const e1 = {
+                target: button1,
+                type: 'click',
+            }
+            autocapture._captureEvent(e1, lib)
+            const props1 = getCapturedProps(lib.capture)
+            expect(props1['$elements'][0]).toHaveProperty('$el_text')
+            expect(props1['$elements'][0]['$el_text']).toMatch('')
+        })
     })
 
     describe('_addDomEventHandlers', () => {
+        const sandbox = sinon.createSandbox()
```

Comment:
Note - you're using old-style (non-jest) mocking - this was probably way harder than it needed to be. Doesn't block this PR though

Reply:
oh right - I just leveraged existing tests as a template